torch/_dynamo/variables/torch.py

```diff
@@ -647,6 +647,8 @@ def map_fn(v):
                     return v
 
             return torch.utils._pytree.tree_map(map_fn, tree)
+        elif isinstance(self.value, types.ModuleType):
+            raise TypeError("'module' object is not callable")
```

Comment:
Can you please provide a test case? I wonder if we could catch this sooner.

Reply:
sure. umm, do we have good infra for testing assertion messages?  i know in my case it raises the TypeError  wrapped in some dynamo wrapper, so that might be annoying to 'match' to the eager behavior but hopefully we already deal with that in some harness?